src/js/dataMapping/covid19/recipient/recipient.jsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * recipient.js
```

Comment:
Can the file extension change from `.jsx` --> `.js`?

Reply:
Not with want I want to do in that file :). Moved it to the containers helpers file.